astrapy/db.py

```diff
@@ -825,6 +873,29 @@ def upsert(self, document: API_DOC) -> str:
 
         return upserted_id
 
+    def batched_concurrent_upsert(
```

Comment:
Name it `upsert_many` ?

Reply:
My only issue with that is, we chose insert_many because it's an API call. Here we're following the logic of basically wrapping the api call and producing chunks. So maybe chunked_upsert? to fit the convention of the other?